test/index.js

```diff
@@ -96,7 +96,7 @@ describe("depcheck", function () {
 
     depcheck(absolutePath, {  }, function checked(unused) {
       assert.equal(unused.dependencies.length, 1);
-      //assert.deepEqual(Object.keys(unused.invalidFiles).length, 2);
+      assert.deepEqual(Object.keys(unused.invalidFiles).length, 2);
```

Comment:
I don't think the `depcheck-es6/test/fake_modules/bad_js/deeper/also_bad_js.js` file is necessary. Please remove it and change the number to 1.

Besides, could you please add more assertion on `invalidFiles`. For example, from the `Object.keys(..).length` assertion, user cannot get the shape of the object shape of `invalidFiles` object. The test should have the ability to do so.